### PR TITLE
[WIP] Implement join action

### PR DESCRIPTION
**Pull request by ml7757**
Implement actions to allow players to join a game. Still work in process.

### CODE REVIEW

**Comment by ml7757** [comment]
button-label

**Comment by ml7757** [comment]
Will now redirect to individual game page:

before

after

File: src/containers/Lobby.js [hunk 1]
```diff
@@ -37,7 +42,19 @@ class Lobby extends PureComponent {
                 <Paper
                   zDepth={1}
                   style={{ padding: '12px 24px' }}>
-                  <h4>{ game.title }</h4>
+                    <h4>{ game.title }</h4>
+                    <div>
+                      { game.playerIds.length < 2 &&
```
Comment (Woulei) [inline src/containers/Lobby.js:47]: Just a question. `game.playerIds.length < 2 &&` what? what else are you comparing against?

**Review by Woulei — APPROVED** [review]
Just one comment. But it still works so..
🚢

**Pull request merged**